### PR TITLE
Ratings and reviews - ready for deployment

### PR DESCRIPTION
**Pull request by melissa-gv**

### CODE REVIEW

File: src/components/PhotoUploadWidget.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    },
+      (error, result) => {
+        if (!error && result && result.event === "success") {
+          this.uploadedPhotoURLs.push(result.info.url)
+          this.uploadedPhotoThumbnailURLs.push(result.info.thumbnail_url)
+          this.props.handlePhotoUploadResponse(this.uploadedPhotoURLs, this.uploadedPhotoThumbnailURLs)
+        }
+      }
+    );
+    document.getElementById("upload_widget").addEventListener(
```
Comment (AllEyesBlank) [inline src/components/PhotoUploadWidget.jsx:58]: also the event listener format here is not using React-- I don't think it necessarily matters (and I used this format when I edited the index.html in order to implement dark mode) but I'm just wondering why the divergence in this case.

File: src/index.jsx [hunk 2]
```diff
@@ -45,21 +45,23 @@ class App extends React.Component {
       console.log(err));
   }
 
-  selectProduct (product) {
+
+
+  selectProduct(product) {
+    console.log('selectProduct was clicked!')
```
Comment (AllEyesBlank) [inline src/index.jsx:51]: We should probably get rid of this log.

File: src/components/Stars.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      }
+    }
+    return starsOverlay;
+  }
+
+
+  return (
+    <span className='stars-component'>
+      {starsUnderlay}
+      {getStarsIcons()}
```
Comment (AllEyesBlank) [inline src/components/Stars.jsx:34]: This is super clever implementation! Well done!

File: src/components/RatingsReviews/RatingsReviews.jsx [hunk 4]
```diff
@@ -13,23 +13,26 @@ class RatingsReviews extends React.Component {
...
   }
 
   componentDidUpdate(prevProps) {
     if (this.props.product !== prevProps.product) {
       axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/reviews', {
         headers: {'Authorization': `${API_KEY}`},
         params: {
-          count: 2,
-          page: 1,
+          count: 50,
```
Comment (AllEyesBlank) [inline src/components/RatingsReviews/RatingsReviews.jsx:29]: thoughtful to store the reviews on the first get request... i like reducing the time complexity like that, great job!

**Review by AllEyesBlank — COMMENTED** [review]
This all looks really good, Melissa! Just a few minor quibbles-- the main thing is the upload widget seems pretty stylistically disparate from the rest of the app. I think that's fine -- maybe the code is from somewhere else? -- so you don't need to change it since our deadline is so close. But worth thinking about how some of these things could be done with our stylesheet and using React instead!

File: src/index.jsx (hunk 2 above)
Comment (melissa-gv) [reply src/index.jsx:51]: Agreed, I'll remove it!

**Pull request merged**